frontend/src/components/v1/AlertBanner.css

```
@@ -0,0 +1,322 @@
+  padding: 0.875rem 1.25rem;
+  border-radius: 8px;
+  font-family: var(--font-family, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif);
+  font-size: 0.875rem;
+  line-height: 1.5;
+  transition: all 0.2s cubic-bezier(0.16, 1, 0.3, 1);
+  animation: alertBannerFadeIn 0.35s cubic-bezier(0.16, 1, 0.3, 1);
+  box-shadow: var(--shadow-sm, 0 1px 2px 0 rgba(0, 0, 0, 0.05));
+  margin-bottom: 1rem;
+}
+
+@keyframes alertBannerFadeIn {
```

Stylelint (17.13.0)

[error] Expected keyframe name "alertBannerFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)